src/common/components/button/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FunctionComponent, ReactElement } from 'react';
+import joinClassNames from 'classnames';
+
+import styles from './button.module.scss';
+
+const Button: FunctionComponent<{
+  className?: string;
+  children: string | ReactElement;
```

Comment:
How about?
```suggestion
  children: React.ReactNode;;
```

Reply:
i want vanilla `<button>foo</button>` work here too, not stricted to only jsx

Reply:
Yes, as far as I know, `ReactNode` includes strings, nulls, JSX elements, etc.

Reply:
ah, i'll remove it then

Reply:
`Type 'string' is not assignable to type 'ReactElement<any, string | ((props: any) => ReactElement<any, string | ... | (new (props: any)`

Reply:
fixed